src/app/views/homepage/edit/EditHomepage.jsx

```diff
@@ -1,77 +1,86 @@
-import React, { Component } from "react";
+import React from "react";
 import PropTypes from "prop-types";
 import SimpleEditableList from "../../../components/simple-editable-list/SimpleEditableList";
 import SaveAndReviewActions from "../../../components/save-and-review-actions/SaveAndReviewActions";
 import Input from "../../../components/Input";
 
 import { connect } from "react-redux";
 
-class EditHomepage extends Component {
-    render() {
-        return (
-            <div className="grid__col-6 margin-bottom--4">
-                <div className="margin-top--2">
-                    &#9664;{" "}
-                    <button type="button" className="btn btn--link" onClick={this.props.handleBackButton}>
-                        Back
-                    </button>
-                </div>
-                <h1 className="margin-top--1 margin-bottom--1">Edit Homepage</h1>
-                <h2 className="margin-top--0">Headlines</h2>
-                <SimpleEditableList
-                    addText={"Add headline"}
-                    fields={this.props.homepageData.featuredContent}
-                    editingStateFieldName="featuredContent"
-                    handleAddClick={this.props.handleSimpleEditableListAdd}
-                    handleEditClick={this.props.handleSimpleEditableListEdit}
-                    handleDeleteClick={this.props.handleSimpleEditableListDelete}
-                    maximumNumberOfEntries={this.props.maximumNumberOfEntries}
-                    disableActions={this.props.disableForm}
-                />
-                <h2 className="margin-top--1">Around ONS</h2>
-                <SimpleEditableList
-                    addText={"Add around ONS feature"}
-                    fields={this.props.homepageData.aroundONS}
-                    editingStateFieldName="aroundONS"
-                    handleAddClick={this.props.handleSimpleEditableListAdd}
-                    handleEditClick={this.props.handleSimpleEditableListEdit}
-                    handleDeleteClick={this.props.handleSimpleEditableListDelete}
-                    maximumNumberOfEntries={this.props.maximumNumberOfEntries}
-                    disableActions={this.props.disableForm}
-                />
-                <h2 className="margin-top--1">Service Message</h2>
-                <Input
-                    id="serviceMessage"
-                    label=""
-                    type="textarea"
-                    value={this.props.homepageData.serviceMessage}
-                    onChange={this.props.handleStringInputChange}
-                    disabled={this.props.disableForm}
+const EditHomepage = ({
```

Comment:
Do we need to define all of the props here?
`const EditHomepage = props => (...`
I think this is where propTypes comes in to help define what are valid props for this component? Or is it better practice to use both?

Reply:
I prefer destructing props so in this case as they are used they are needed :-)